community-expectations.md

```diff
@@ -0,0 +1,85 @@
+## Community Expectations
+
+Kubernetes is a community project and, as such, it is wholly dependent on
+its community to provide a productive, friendly and collaborative environment.
+
+The first and foremost goal of the Kubernetes community to develop orchestration
+technology that radically simplifies the process of creating reliable
+distributed systems, however a second, equally important goal is the creation
+of a community that fosters easy, agile development of such an orcheastration
+systems.
+
+This document describes the community expectations for
+members of the Kubernetes community.  It is intended to be a living document
+that evolves as the community evolves.  It currently covers the expectations
+of conduct that govern all members of the community as well as the expectations
+around code review that govern all active contributors to Kubernetes.
+
+### Code of Conduct
+
+The most important expectation of the Kubernetes community is that all members
+abide by the Kubernetes [community code of conduct](code-of-conduct.md).
+Only by respecting each other can we develop a productive, collaborative
+community.
+
+### Code review
+
+As a community we believe in the value of code review for all contributions.
+Code review increases both the quality and readability of our codebase, which
+in turn produces high quality software.
+
+However, the code review process can also introduce latency for contributors
+and additional work for reviewers that can frustrate both parties.
+
+Consequently, as a community we expect that all active participants in the
+community will also be active reviewers.  To that end, we have set out the
+following expectations for code review:
+
+Any *active contributor* to the Kubernetes project is expected to be available
+to review code in areas where that contributor has expertise.  Active
+contributors or considered to be anyone who meets any of the following criteria:
+   * Sent more than two pull requests (PRs) in the previous one month, or more
```

Comment:
Agree with Clayton, that bar is too low.

Also, are we committing to tracking this?  Who is going to write the bot that adds and removes people from the reviewer pool?

I'm really uncomfortable with the numbers.  I am worried that, as soon as we start assigning reviews, this could be chilling.  Especially if the bar is low.  Someone who fixes typos, adds comments, clarifies errors, etc could EASILY hit 2 or 4 PRs in a month.  Someone who does some testing for their deployment could easily file 3 or more issues.  Commenting on PRs is cheap - do we want to discourage that?

This still feels like negative reinforcement to me, and we all know that doesn't work long term.  "If you want to play you have to review".  Can't we twist it around and just get the ladder defined - if you do X you are a "star reviewer"r.  If you do Y (> X), you are a "maintainer".  If you do Z (> Y) you are an "owner".  Positive reinforcement.